conf/metrics.conf

```diff
@@ -833,8 +833,9 @@ metric {
 
     group "phishing" {
         symbol "PHISHING" {
-            weight = 3.0;
```

Comment:
I'll leave it up to you.
Without `one_shot = true;` I had sometimes more than 12. Considering nobody has complained about it for a long time I think `6.0` should be safe, at least it is better than it was. On the other hand a lot of legitimate mail use redirectors, so I understand your concern.